tests/e2e/test_load.py

```diff
@@ -0,0 +1,77 @@
+import os
+import random
+import string
+
+import pytest
+import torch
+import ssl
+import functools
+import certifi
+from azstoragetorch.io import BlobIO
+from azure.identity import DefaultAzureCredential
+from azure.storage.blob import BlobServiceClient
+
+
+@pytest.fixture(scope="module")
+def model():
+    ssl._create_default_https_context = functools.partial(
+        ssl.create_default_context, cafile=certifi.where()
+    )
```

Comment:
For this line, we should probably do the following:

1. Move it out to it's own fixture. We can give it the name `ssl_inject_certifi`
2. In the fixture, use [`monkeypatch`](https://docs.pytest.org/en/stable/how-to/monkeypatch.html) built-in fixture to make sure we restore state back to the original value. Mainly in doing this we are mutating global state and when global state is mutated it can impact logic later on and hard to tell why it is being impacted in a particular way. So we want to be careful whenever we do things like this.
3. Add a comment in the fixture on why we need this. Mainly `torch.hub` does not use `certifi` which means it delegates to system certificates and some environments may run into SSL errors without it.